src/main/java/ua/softserveinc/tc/util/DateUtil.java

```diff
@@ -41,6 +43,10 @@ public static Date toDate(String date) {
         }
     }
 
+    public static Date toDate(LocalDate date) {
```

Comment:
I need to convert i LocalDate to Date, in this class we already have some methods: toDate(String date) etc, so I just convert LocalDate to Date is it misleading?
to convert LocalDate to Date I find this way to covert it: http://www.java2s.com/Tutorials/Java/Data_Type_How_to/Date_Convert/Convert_LocalDate_to_java_util_Date.htm